core/src/main/scala/chisel3/internal/firrtl/Converter.scala

```diff
@@ -24,6 +24,12 @@ private[chisel3] object Converter {
     case Percent => ("%%", List.empty)
   }
 
+  def getRef(id: HasId, sourceInfo: SourceInfo): Arg =
+    id.getOptionRef.getOrElse(
+      throwException(s"Internal error! Could not get ref for $id!" +
+        sourceInfo.makeMessage(" See " + _))
+    )
```

Comment:
Does this error message include some stuff about this being an actual bug and to report it? If not, should it?

Reply:
Yeah it probably should, I'll make the error message more verbose.